### ISSUE TITLE
No images in readme

### ISSUE THREAD

**Issue by ErikReider** [nwg-piotr/nwg-dock#23]
Just a quickie. Looks like the linked images in the readme are down

**Comment by nwg-piotr** [comment]
Yes, in all my repos. The scrot.cloud service is gone. I'll have to add every singe image again.
